ruleset/rules/0755-office365_rules.xml

```diff
@@ -0,0 +1,928 @@
+    </rule>
+
+    <rule id="91580" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^52$</field>
+        <description>Office 365: Data Insights REST API events.</description>
+        <options>no_full_log</options>
+        <group>DataInsightsRestApiAudit</group>
+    </rule>
+
+    <rule id="91581" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^53$</field>
+        <description>Office 365: Events related to the application of information barrier policies.</description>
+        <options>no_full_log</options>
+        <group>InformationBarrierPolicyApplication</group>
+    </rule>
+
+    <rule id="91582" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^54$</field>
+        <description>Office 365: SharePoint list item events.</description>
+        <options>no_full_log</options>
+        <group>SharePointListItemOperation</group>
+    </rule>
+
+    <rule id="91583" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^55$</field>
+        <description>Office 365: SharePoint list content type events.</description>
+        <options>no_full_log</options>
+        <group>SharePointContentTypeOperation</group>
+    </rule>
+
+    <rule id="91584" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^56$</field>
+        <description>Office 365: SharePoint list field events.</description>
+        <options>no_full_log</options>
+        <group>SharePointFieldOperation</group>
+    </rule>
+
+    <rule id="91585" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^57$</field>
+        <description>Office 365: Teams admin events.</description>
+        <options>no_full_log</options>
+        <group>MicrosoftTeamsAdmin</group>
+    </rule>
+
+    <rule id="91586" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^58$</field>
+        <description>Office 365: Events related to HR data signals that support the Insider risk management solution.</description>
+        <options>no_full_log</options>
+        <group>HRSignal</group>
+    </rule>
+
+    <rule id="91587" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^59$</field>
+        <description>Office 365: Teams device events.</description>
+        <options>no_full_log</options>
+        <group>MicrosoftTeamsDevice</group>
+    </rule>
+
+    <rule id="91588" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^60$</field>
+        <description>Office 365: Teams analytics events.</description>
+        <options>no_full_log</options>
+        <group>MicrosoftTeamsAnalytics</group>
+    </rule>
+
+    <rule id="91589" level="5">
```

Comment:
I think this level should be higher since these alerts are from compromised users.

Reply:
Done